### DISCUSSION TITLE
Pihole: General error connecting with API

### DISCUSSION THREAD

**Discussion by boehser-enkel** [linuxserver/Heimdall#1005] [App issues]
Pihole: General error connecting with API
Get this error since updating pihole to the latest version

**Comment by riyad** [comment]
Looking into the network tab of my browser I also see 500 errors for the Pihole status requests.

The server responses look like this:
```json
{
    "message": "Undefined variable: ads_blocked_today (View: /config/www/SupportedApps/Pihole/livestats.blade.php)",
    "exception": "Illuminate\\View\\ViewException",
    "file": "/var/www/localhost/heimdall/storage/framework/views/41a5224208650f9c14c769fd232f332b9fcaa019.php",
    "line": 4,
    "trace": [...]
}
```

If required I can add the full trace also.

**Comment by epiech** [comment]
Has there been any update on this issue? I don't see the error on my Pihole bare metal install on a Raspberry Pi4 but I get tthis error on my Docker container hosted on unRAID.
